azure-pipelines.yml

```diff
@@ -52,9 +52,9 @@ stages:
     - script: make  EXTRA_FLAGS=-Werror test-all
       displayName: 'Run all unit tests'
     - script: make  EXTRA_FLAGS=-Werror all
-      displayName: 'Build all oficial targets'
+      displayName: 'Build all official targets'
     - script: mkdir release; cp obj/*.hex release/
-      displayName: 'Copy artefacts'
+      displayName: 'Copy artifacts'
```

Comment:
This is an interesting one - while 'artefact' is correct when discussing signal forms, 'artifact' is indeed the correct one when discussing things spit out by a CI pipeline.